### PR TITLE
Mov platforms fix

### PR DESCRIPTION
**Pull request by Jayd3nPye**
## What Changed ##
- Tweaks to ranged attack
- Tweaks to zoom in/out
- Slight tweaks to player collision
- Fixed avalanche not dealing damage on level 6
- Added animated and unanimated death depending on how you die (Unanimated death only happens if you fall into the void)

This is most likely my final pr for the year so I hope everything is fixed!

### CODE REVIEW

**Comment by Jayd3nPye** [comment]
Oh yeah I forgot to say that I fixed the moving platforms so they don't experience collision with the player

**Comment by Jayd3nPye** [comment]
I hope everything works now ;-;

File: src/global/main.gd [hunk 1]
```diff
@@ -1,34 +1,59 @@
...
 var suffixes
+
+#Avalanche hit detector
 var snowHit = 0
-var leaderboardOffer = false
+
+#Player mov suspended detector
+var suspendMovement = false
+
+#No Idea/ -- Robbin Please explain --
```
Comment (Newroll) [inline src/global/main.gd:42]: this just the code for the player going flash flash and immunity, don't worry abt it

File: src/global/main.gd [hunk 2]
```diff
@@ -1,34 +1,59 @@
...
-var suspendMovement = false
+
+#Camera Zoom Levels
+var zoomLevels = [0.5, 0.75, 1, 1.25, 1.5]
+var currentZoomLevel = 2
 
 func _physics_process(_delta):
 	
+	
+	#Does this need to be removed?
```
Comment (Newroll) [inline src/global/main.gd:54]: no leave this in

**Review by Newroll — APPROVED** [review]
lgtm, please keep bug testing on main brunch, report asap if you find any.

we'll have a chat tmr morning abt the demo lvl and how we want it done.

**Pull request merged**